src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    flex-wrap: wrap;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background: url(../images/star.svg) no-repeat center;
+  }
+
+  &__star:last-of-type {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point